db/schema.rb

```diff
@@ -256,7 +257,7 @@
     t.index ["pilot_flight_id"], name: "index_pf_results_on_pilot_flight_id"
   end
 
-  create_table "pfj_results", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci" do |t|
+  create_table "pfj_results", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci" do |t|
```

Comment:
This is one table whose records get "rolled" (deleted and created) on recomputation. It still has quite a way to go before encountering the integer count limit.
```
mysql> select max(id) from pfj_results;
+---------+
| max(id) |
+---------+
|  234905 |
+---------+
1 row in set (0.00 sec)
```